app/Models/Ticket.php

```diff
@@ -25,10 +25,10 @@ public function rules()
         return [
             'title'              => 'required',
             'price'              => 'required|numeric|min:0',
-            'description'        => '',
-            'start_sale_date'    => 'date_format:"'.$format.'"',
-            'end_sale_date'      => 'date_format:"'.$format.'"|after:start_sale_date',
-            'quantity_available' => 'integer|min:'.($this->quantity_sold + $this->quantity_reserved)
+            'description'        => 'nullable',
```

Comment:
These should not be nullable to be honest. It's the data that drive this system mainly and we want to constrain as much of it as possible. Please add default values for these as well as in the migrations?

Reply:
The 'start_sale_date', 'end_sale_date' and 'quantity_available' fields are configured as nullable in your first migration.

I have simply updated the rules to reflect the migration, I have created a new migration to update the default value of the description.

To put the others as "non nullable" I think you should check the code in case we break something, if you can check it before you merge this better.